app/src/main/java/org/openobservatory/ooniprobe/common/ResubmitTask.java

```diff
@@ -122,7 +122,7 @@ protected Boolean doInBackground(Integer... params) {
     protected void onPostExecute(Boolean result) {
         super.onPostExecute(result);
         A activity = getActivity();
-        if (activity != null) {
+        if (activity != null  && result) {
```

Comment:
shouldn't this be more readable as `result != null`? Can you also avoid adding two spaces before `&&`?

Reply:
I need to check the result is true, not null

Reply:
My apologies, then :whale: